File: lista_laco_repeticao/exercicio3.py
print("Este programa exibe uma tabuada deste número")
numero = int(input("Digite um número inteiro positivio: "))
tabuada = 0

for i in range (0, 10):
    tabuada= tabuada+1
    resultado = numero*tabuada
    print(f"{numero} x {tabuada} = {resultado}")

File: lista_laco_repeticao/exercicio3.php
<?php

echo "Este programa exibe uma tabuada deste número\n";
$numero = (int)readline("Digite um número inteiro positivio: ");
$tabuada = 0;

for ($i = 0; $i < 10; $i++) {
    $multiplicador = $i + 1; 
    $resultado = $numero * $multiplicador;
    echo "$numero x $multiplicador = $resultado\n";
}

?>